src/bin/compute-wer.cc

```diff
@@ -91,36 +119,26 @@ int main(int argc, char *argv[]) {
       }
       num_words += ref_sent.size();
       int32 ins, del, sub;
-      word_errs += LevenshteinEditDistance(ref_sent, hyp_sent, &ins, &del, &sub);
+      int32 this_word_errs = LevenshteinEditDistance(ref_sent, hyp_sent, &ins, &del, &sub);
+      word_errs += this_word_errs;
       num_ins += ins;
       num_del += del;
       num_sub += sub;
 
       num_sent++;
-      sent_errs += (ref_sent != hyp_sent);
+      int32 this_sent_errs = (ref_sent != hyp_sent);
```

Comment:
since you're changing this, would be easier to do:
if (ref_sent != hyp_sent)
   sent_errs++;